code-challenges.test.js

```diff
@@ -0,0 +1,99 @@
+// ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
+
+// Please read all questions thoroughly
+// Pseudo coding is REQUIRED
+// If you get stuck, please leave comments to help us understand your thought process
+
+// Use test driven development to complete the following questions
+// Add appropriate dependencies: $ yarn add jest
+
+// Reminder: The test will call your function
+// Run the file with the following command: $ yarn jest
+
+// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+
+// I shall call it chopShuffle. the name is self explanitory.
+
+// a) Create a test with an expect statement using the variable provided.
+// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+
+// After checking out the link i decided to create the testing perameters.
+
+describe("chopShuffle", () => {
+  //moved the variable into the testing perameters.
+  const colors1 = ["purple", "blue", "green", "yellow", "pink"];
+  it("removes the first item from the array and shuffles the remaining content", () => {
+    // After research, i noticed .toContain will check to see if the array contains any values i wish to expect in no particular order. i thought this is a good idea since the order is going to be randomized.
+    expect(chopShuffle(colors1)).toContain("yellow", "blue", "pink", "green");
```

Comment:
I would recommend using expect.arrayContaining to check the array itself.  .toContain will only check for one instance in the array, not the entire array.

Reply:
interesting. what do you mean by one instance?